libcloud/compute/drivers/outscale.py

```diff
@@ -412,23 +457,80 @@ def create_node(self,
             data.update({"SubnetId": ex_subnet_id})
         action = "CreateVms"
         data = json.dumps(data)
-        return self._to_node(self._call_api(action, data).json()["Vms"][0])
+        node = self._to_node(self._call_api(action, data).json()["Vms"][0])
+        if name is not None:
+            action = "CreateTags"
+            data = {
+                "DryRun": ex_dry_run,
+                "ResourceIds": [node.id],
+                "Tags": {
+                    "Key": "Name",
+                    "Value": name
+                }
+            }
+            data = json.dumps(data)
+            if self._call_api(action, data).status_code != 200:
+                return False
+            action = "ReadVms"
+            data = {
+                "DryRun": ex_dry_run,
+                "Filters": {
+                    "VmIds": [node.id]
+                }
+            }
+            return self._to_node(
+                self._call_api(action, json.dumps(data)).json()["Vms"][0]
+            )
+        return node
 
     def reboot_node(self, node: Node):
         """
         Reboot instance.
 
-        :param      node: the ID(s) of the VM(s)
-                    you want to reboot (required)
+        :param      node: VM(s) you want to reboot (required)
         :type       node: ``list``
 
         :return: the rebooted instances
         :rtype: ``dict``
         """
         action = "RebootVms"
-        data = json.dumps({"VmIds": node.id})
+        data = json.dumps({"VmIds": [node.id]})
+        if self._call_api(action, data).status_code == 200:
+            return True
+        return False
+
+    def start_node(self, node: Node):
+        """
+        Start a Vm.
+
+        :param      node: the  VM(s)
+                    you want to start (required)
+        :type       node: ``Node``
+
+        :return: the rebooted instances
+        :rtype: ``dict``
+        """
+        action = "StartVms"
+        data = json.dumps({"VmIds": [node.id]})
+        if self._call_api(action, data).status_code == 200:
```

Comment:
Same here.

Just returning ``False`` will make troubleshooting in such scenarios much harder.